pydantic/_internal/_validators.py

```diff
@@ -60,21 +60,12 @@ def sequence_validator(
     value_type = type(__input_value)
     v_list = validator(__input_value)
 
+    if issubclass(value_type, (str, bytes)):
+        raise PydanticCustomError('sequence_str_str_type', 'Strings are not allowed as a Sequence value')
```

Comment:
Should you continue to distinguish between str/bytes here?

Reply:
Yeah, providing two distinct errors seems like a good idea:)

Reply:
It's possible that `value_type` is a subclass of `str` or `bytes`. Thus, it looks like it would be more useful to return the actual type name being used.